Examples/Tests/galilean/analysis.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+"""
+This script is used to test the results of the Galilean PSATD and
+averaged Galilean PSATD methods in WarpX.
+
+It compares the energy of the electric field with a given reference energy.
+
+The reference energy is computed by running the same test with:
+(i)  psatd.v_galilean=(0,0,0) for Galilean tests, or
+(ii) psatd.do_time_averaging=0 for averaged Galilean tests.
+
+In both cases, the reference energy corresponds to unstable results due to NCI
+(suppressed by the Galilean PSATD method, without or with averaging, respectively).
+"""
+import os
+import re
+import sys
+
+import numpy as np
+import scipy.constants as scc
+
+import yt ; yt.funcs.mylog.setLevel(0)
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+filename = sys.argv[1]
+
+# Parse some input arguments from output file 'warpx_used_inputs'
+current_correction = False
+time_averaging = False
+warpx_used_inputs = open('./warpx_used_inputs', 'r').read()
+if re.search('geometry.dims\s*=\s*2', warpx_used_inputs):
+    dims = '2D'
+elif re.search('geometry.dims\s*=\s*RZ', warpx_used_inputs):
+    dims = 'RZ'
+elif re.search('geometry.dims\s*=\s*3', warpx_used_inputs):
+    dims = '3D'
+if re.search('psatd.current_correction\s*=\s*1', warpx_used_inputs):
+    current_correction = True
+if re.search('psatd.do_time_averaging\s*=\s*1', warpx_used_inputs):
+    time_averaging = True
```

Comment:
@dpgrote @ax3l Please feel free to suggest any better way to parse these input parameters from our output file `warpx_used_inputs` (note that the flags set here are used below to set reference energy values and tolerances independently).